components/UI/Drawer/components/channels/babala/FeedChannelBabala.tsx

```diff
@@ -29,15 +26,16 @@ const FeedChannelBabala = ({
       <div style={styles.container}>
         <div style={styles.logo_container}>
           <Typography style={styles.logo}>Babala</Typography>
-          <div style={styles.chip_container}>
-            {isNotNan(reason) && (
-              <Chip label={reason?.toLowerCase()} color="info" />
-            )}
-          </div>
         </div>
         <Divider />
         <Typography style={styles.fullText}>{full_text}</Typography>
         {extraValues}
+
+        <div style={styles.chip_container}>
+          {!isNaN(reason) && reason && (
+            <Chip label={capitalize(reason)} color="info" />
+          )}
+        </div>
```

Comment:
i think we should wrap the outer element with this conditional as well

Reply:
You are right, I'll fix it!